robocoder99/sonic-utilities
# [psushow] Add "HW Rev" field to psushow output

#### What I did

Added a field for hardware revision to the psushow output table. 

#### How I did it

Defined new psu status field sourced from "revision" field in STATE_DB added in https://github.com/robocoder99/sonic-platform-daemons/pull/1 and adds it to the table output with the header "HW Rev"

All relevant unit tests were also updated.

#### How to verify it

Execute `show platform psustatus` and verify that "HW Rev" column is present and appropriately populated depending on the platform. 

#### Previous command output (if the output of a command-line utility has changed)

```
PSU    Model    Serial      Voltage (V)    Current (A)    Power (W)  Status    LED
-----  -------  --------  -------------  -------------  -----------  --------  -----
PSU 1  N/A      N/A               12.08           3.62        43.75  OK        green
```

#### New command output (if the output of a command-line utility has changed)

```
PSU    Model          Serial        HW Rev      Voltage (V)    Current (A)    Power (W)  Status    LED
-----  -------------  ------------  --------  -------------  -------------  -----------  --------  -----
PSU 1  MTEF-PSF-AC-A  MT1621X15246  A3                11.97           4.56        54.56  OK        green
```
